venafi/resource_venafi_certificate.go

```diff
@@ -341,6 +353,22 @@ func enrollVenafiCertificate(d *schema.ResourceData, cl endpoint.Connector) erro
 		}
 	}
 
+	if ttl, ok := d.GetOk("valid_days"); ok {
+
+		validDays := ttl.(int)
+		validDays = validDays * 24
+		expiration_window := d.Get("expiration_window").(int)
+
+		if validDays < expiration_window {
+			validDays = expiration_window
+		}
+
+		req.ValidityHours = validDays
```

Comment:
Confusing variable naming... looks like a logic error because you're assigning a days value to an hours variable.  Maybe just renaming `validDays` in this code block to `validity`?

Reply:
fixed on most recent commit.